### PR TITLE
Dialog bw theme

### PR DESCRIPTION
**Pull request by jonathanadler**
Work in progress.

Currently the Text and Button elements were added a black & white theme.
You can see it working in the settings panel.

Which components should we upgrade next?

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_530/

File: src/components/Dialog/Dialog.tsx [hunk 1]
```diff
@@ -82,28 +83,32 @@ export class Dialog extends React.Component<DialogProps> {
...
-                    data-hook={DATA_HOOKS.CLOSE_BTN}
-                    aria-label={closeButtonAriaLabel}
-                    aria-labelledby={closeButtonAriaLabelledby}
-                    innerRef={closeButtonRef}
-                    onClick={onClose}
-                    icon={<CloseIcon />}
-                  />
+                <div
+                  className={`${classes.contentWrapper} ${contentClassName ||
+                    ''}`}
```
Comment (sivanharel) [inline src/components/Dialog/Dialog.tsx:91]:
```suggestion
              className={st(classes.contentWrapper, contentClassName)}  ''}`}
```

Comment (sivanharel) [inline src/components/Dialog/Dialog.tsx:91]:
```suggestion
                  className={st(classes.contentWrapper, contentClassName)}
```

Comment (sivanharel) [reply src/components/Dialog/Dialog.tsx:91]: I know that it something that I wrote but it's better to use the st method :)

File: src/components/Dialog/docs/DialogWiringExample.tsx [hunk 2]
```diff
@@ -50,7 +49,32 @@ export class DialogWiringExample extends React.Component {
           onClose={this._onCloseDialog}
           wiredToSiteColors={isWired}
         >
-          <div className={classes.content}>This is the content!</div>
+          <div className={classes.content} style={{ textAlign: 'center' }}>
```
Comment (sivanharel) [inline src/components/Dialog/docs/DialogWiringExample.tsx:52]: It seems like the style={{ textAlign: 'center' }} is not needed, it's defined in the css file as well :)

**Pull request closed without merging**